assets/js/hocs/test/with-attributes.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
```

Comment:
There's [some debate about moving away from enzyme](https://github.com/WordPress/gutenberg/issues/17249) in the core GB project (in particular the usage of shallow).  I think there's some valid reasons to consider that for our project here as well.  Something to consider.  I noticed you used `react-test-renderer` in the `withCategories` hoc test, why the decision to use enzyme here?

Reply:
Interesting, I didn't know about that. I replaced enzyme in 107e163abc7314f26c79c5d8f87c3ba91d224863.